CHANGELOG.md

```diff
@@ -7,8 +7,8 @@ Full documentation for hipSOLVER is available at [hipsolver.readthedocs.io](http
 - Added compatibility API with hipsolverSp prefix
 - Added compatibility-only functions
   - csrlsvchol
-    - hipsolverSpScsrlsvcholHost, hipsolverSpDcsrlsvcholHost, hipsolverSpCcsrlsvcholHost, hipsolverSpZcsrlsvcholHost
```

Comment:
The documentation for CHOLMOD mentions it has support for two different storage formats for complex matrices, one is the traditional interleave of real and imag parts, another format separately stores all real part in one matrix, and all imaginary part in another matrix.

Reply:
I'm leaving out complex precision support from the current release because https://github.com/ROCmSoftwarePlatform/rocSOLVER/pull/582 is also leaving out complex support. I'm hoping that it won't be too difficult to get the complex methods working using SuiteSparse once it comes time to add them in.